### PR TITLE
Updated views section

### PR DESCRIPTION
**Pull request by kamil-kaczmarek**

### CODE REVIEW

**Comment by kamil-kaczmarek** [comment]
Hey @jakubczakon 

Updated view section. I also updated customizing columns section as it was important for the views.

Ready for review. Only loom not recorded yet.

File: docs/organizing-and-exploring-results-in-the-ui/organizing-experiments.rst [hunk 1]
```diff
@@ -28,37 +27,182 @@ You can create tag(s), which you assign to experiments. Later, you can quickly f
...
+Moreover, you can reorder columns and adjust their width.
+
+.. image:: ../_static/images/organizing-and-exploring-results-in-the-ui/experiment-dashboard/col-order-width.gif
+   :target: ../_static/images/organizing-and-exploring-results-in-the-ui/experiment-dashboard/col-order-width.gif
+   :alt: columns order and width
 
 Grouping experiments
 --------------------
-
 You can group experiments by one or more column(s). The dashboard displays the selected columns, allowing you to make in-group and across-groups analysis of the experiments. Each group is represented by the first experiment that appears according to the sorting order. After opening it, each group shows at most 10 experiments - all experiments can be viewed by clicking **Show all**.
```
Comment (jakubczakon) [inline docs/organizing-and-exploring-results-in-the-ui/organizing-experiments.rst:175]: I'd add space after some sentences here -> it is a big text blob right now.

**Review by jakubczakon — CHANGES_REQUESTED** [review]

File: docs/organizing-and-exploring-results-in-the-ui/organizing-experiments.rst [hunk 2]
```diff
@@ -18,7 +18,6 @@ There are several ways to organize your experiments.
 
 Using tags
```
[the remark this answers is not in the document]
Comment (kamil-kaczmarek) [reply docs/organizing-and-exploring-results-in-the-ui/organizing-experiments.rst:19]: what options you have in mind? I did not work with tags section.

File: docs/organizing-and-exploring-results-in-the-ui/organizing-experiments.rst [hunk 3]
```diff
@@ -28,37 +27,182 @@ You can create tag(s), which you assign to experiments. Later, you can quickly f
 
 Using dashboard views
 ---------------------
+You can create a custom view of the experiments dashboard, that is saved setup of experiments filter and arrangement of columns.
```
[the remark this answers is not in the document]
Comment (kamil-kaczmarek) [reply docs/organizing-and-exploring-results-in-the-ui/organizing-experiments.rst:30]: corrected.

File: docs/organizing-and-exploring-results-in-the-ui/organizing-experiments.rst [hunk 4]
```diff
@@ -28,37 +27,182 @@ You can create tag(s), which you assign to experiments. Later, you can quickly f
...
+.. note::
+
+    Every saved view is visible for everybody in the project.
+
+When dashboard views are useful?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+There are few situation when you may want to create custom dashboard view:
+
+* You work on separate idea or task within a project, and you want to see only relevant information.
+* Project team explores separate ideas in the project and for each idea team wants to have separate dashboard.
```
[the remark this answers is not in the document]
Comment (kamil-kaczmarek) [reply docs/organizing-and-exploring-results-in-the-ui/organizing-experiments.rst:53]: ok

File: docs/organizing-and-exploring-results-in-the-ui/organizing-experiments.rst [hunk 5]
```diff
@@ -28,37 +27,182 @@ You can create tag(s), which you assign to experiments. Later, you can quickly f
...
+* You want to create separate view that contains only your experiments.
+* You want to have separate view for experiments that serve some special purpose, for example experiments that have model pushed to production.
```
[the remark this answers is not in the document]
Comment (kamil-kaczmarek) [reply docs/organizing-and-exploring-results-in-the-ui/organizing-experiments.rst:55]: ok

[the remark this answers is not in the document]
Comment (kamil-kaczmarek) [reply docs/organizing-and-exploring-results-in-the-ui/organizing-experiments.rst:55]: ok

File: docs/organizing-and-exploring-results-in-the-ui/organizing-experiments.rst [hunk 6]
```diff
@@ -28,37 +27,182 @@ You can create tag(s), which you assign to experiments. Later, you can quickly f
...
+
+What can be customized in view?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Generally view is a saved setup of experiments filter and arrangement of columns.
```
[the remark this answers is not in the document]
Comment (kamil-kaczmarek) [reply docs/organizing-and-exploring-results-in-the-ui/organizing-experiments.rst:59]: ok

File: docs/organizing-and-exploring-results-in-the-ui/organizing-experiments.rst [hunk 7]
```diff
@@ -28,37 +27,182 @@ You can create tag(s), which you assign to experiments. Later, you can quickly f
...
+
+Experiments filter, either basic or advanced, can be saved in view. Learn more about it here: :ref:`Searching and filtering experiments <searching-and-filtering-experiments>`.
```
[the remark this answers is not in the document]
Comment (kamil-kaczmarek) [reply docs/organizing-and-exploring-results-in-the-ui/organizing-experiments.rst:61]: ok

File: docs/organizing-and-exploring-results-in-the-ui/organizing-experiments.rst [hunk 8]
```diff
@@ -28,37 +27,182 @@ You can create tag(s), which you assign to experiments. Later, you can quickly f
...
+
+Every setup of columns can be saved in view. Check section :ref:`customizing columns <customizing-columns>` below to learn more about it.
+
+How to create dashboard view?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+loom-placeholder
+
+This short tutorial you will learn how to customize experiment dashboard and save it as a new view.
```
[the remark this answers is not in the document]
Comment (kamil-kaczmarek) [reply docs/organizing-and-exploring-results-in-the-ui/organizing-experiments.rst:69]: ok

File: docs/organizing-and-exploring-results-in-the-ui/organizing-experiments.rst [hunk 9]
```diff
@@ -28,37 +27,182 @@ You can create tag(s), which you assign to experiments. Later, you can quickly f
...
+
+.. note::
+
+    To save view, you need to be project's "`contributor`" or "`owner`". Learn more about it here: :ref:`Roles in project <administration-user-roles-project>`.
+
+Step 1: Go to experiment dashboard
+""""""""""""""""""""""""""""""""""
+Open experiments dashboard in your project. In this tutorial we use |example-project|.
+
+Step 2 and 3 below can be done in any order.
```
[the remark this answers is not in the document]
Comment (kamil-kaczmarek) [reply docs/organizing-and-exploring-results-in-the-ui/organizing-experiments.rst:79]: ok

File: docs/organizing-and-exploring-results-in-the-ui/organizing-experiments.rst (hunk 1 above)
Comment (kamil-kaczmarek) [reply docs/organizing-and-exploring-results-in-the-ui/organizing-experiments.rst:175]: ok

File: docs/organizing-and-exploring-results-in-the-ui/organizing-experiments.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (kamil-kaczmarek) [reply docs/organizing-and-exploring-results-in-the-ui/organizing-experiments.rst:19]: Ok I see

[the remark this answers is not in the document]
Comment (kamil-kaczmarek) [reply docs/organizing-and-exploring-results-in-the-ui/organizing-experiments.rst:19]: corrected

**Comment by kamil-kaczmarek** [comment]
Hey @jakubczakon,

Applied all corrections and added loom video.

Ready for review and please merge if accepted :+1:

**Review by jakubczakon — APPROVED** [review]

**Pull request merged**